cdap-app-fabric/src/test/java/co/cask/cdap/AppWithSchedule.java

```diff
@@ -50,8 +50,11 @@ public void configure() {
       scheduleProperties.put("anotherKey", "anotherValue");
       scheduleProperties.put("someKey", "someValue");
 
-      scheduleWorkflow(Schedules.createTimeSchedule("SampleSchedule", "Sample schedule", "0/1 * * * * ?"),
-                       "SampleWorkflow", scheduleProperties);
+      scheduleWorkflow(Schedules.builder("SampleSchedule")
```

Comment:
I am wondering why the scheduleProperties are separate? Would it be better if they become part of the Schedule and are configured using the schedule builder? Opinions?